app/jobs/crons/schedule_upload_analytics_report_job.rb

```diff
@@ -0,0 +1,8 @@
+class Crons::ScheduleUploadAnalyticsReportJob < CronJob
+  # 10 minutes after GenerateDashboardReportJob
+  self.cron_expression = "10 * * * *"
+
```

Comment:
Shall we add the sentry monitoring line?

Reply:
Yes!